### PR TITLE
ci[python,rust]: Use `env` keyword instead of export

### PR DESCRIPTION
**Pull request by stinodego**
We were already using the `env` keyword in a lot of places - this just make usage consistent across workflows.

Added benefit: this works cross-platform!

### CODE REVIEW

File: .github/workflows/benchmark.yaml [hunk 1]
```diff
@@ -25,20 +25,21 @@ jobs:
...
         with:
           python-version: 3.9
-      - name: Install dependencies
-        shell: bash
-        run: |
-      - uses: r-lib/actions/setup-r@v1
+      - name: Set up R
+        uses: r-lib/actions/setup-r@v1
         with:
-          r-version: '3.5.3' # The R version to download (if necessary) and use.
```
Comment (stinodego) [inline .github/workflows/benchmark.yaml:33]: This was a drive-by change; the 'install dependencies' step doesn't do anything and it breaks the formatting for the rest of the file (although apparently GitHub still parses it).

**Comment by alexander-beedie** [comment]
This ongoing industrialization is good stuff... :)

**Comment by stinodego** [comment]
> This ongoing industrialization is good stuff... :)

There's basically nothing I like to do more 😄
